cli/src/commands/schema/edit.ts

```diff
@@ -362,17 +362,25 @@ export default class EditSchema extends BaseCommand {
 
     try {
       const answer = await snippet.run();
+      const { values } = answer;
```

Comment:
```suggestion
      const { values } = await snippet.run();
```

I don't see more usages. Also I think `enquirer.Snippet` does not return `null | undefined`. It's hard to see because we don't have types for the library, any reason?